_pages/Quotermain.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: Quotermain
+pageUrl: quotermain.github.io
+timestamp: 2018-05-15
+---
```

Comment:
@Quotermain, a newline is really required at the end of the file.  This is why Travis CI builds fails.